packages/svelte-docgen/package.json

```diff
@@ -88,13 +88,13 @@
 	},
 	"dependencies": {
 		"@svelte-docgen/extractor": "workspace:*",
-		"valibot": "catalog:"
+		"valibot": "catalog:",
+		"typescript": "catalog:"
```

Comment:
Hmmm... in this case, the user doesn't need to worry about having TypeScript installed? We will update the version for them, and whenever there are changes in TypeScript libs, e.g. `ESNext`, `DOM`, `DOM.Iterable`, it will be our job to update it.